content/_components/rest-api-component.md

```diff
@@ -8,23 +8,47 @@ The **REST API component** is a simple yet powerful component that allows you to
 
 The REST API component will perform a single REST API call when executed. Incoming data can gets used to configure the API call made and the response from the API call will be the output.
 
-## Introduction
-
-The example below shows the development team creation using the REST API component with our own [REST API service](https://api.{{site.data.tenant.name}}/docs "{{site.data.tenant.name}} REST API service").
-
-![alt text](https://cdn.{{site.data.tenant.name}}/documentation/restapi-component-featuresv2.png "REST API component features")
-*Numbers show: (1) HTTP methods, (2) the URL of the REST API resource, (3) the HTTP call headers and (4) the body of the HTTP request.*
+This document covers the following topics:
 
-1.  REST API component supports the following HTTP methods: `GET`, `PUT`, `POST`, `DELETE` and `PATCH`.
-2.  The URL of the REST API accepts JSONata expressions, meaning the URL address evaluates [JSONata](http://jsonata.org/) expressions.
-3.  Definition of request [headers](#defining-http-headers)
-4.  Definition of request [body](#defining-http-body), if the HTTP method is not `GET`
+*   [Introduction](#introduction)
+*   [Authorisation methods](#authorisation-methods)
+*   [Defining HTTP headers](#defining-http-headers)
+*   [Defining request body](#defining-request-body)
+*   [Working with XML Response](#working-with-xml)
+*   [HTTP Headers in Response](#http-headers)
+*   [Attachments](#attachments)
+*   [Exception handling](#exception-handling)
+*   [Known Limitations](#known-limitations)
 
```

Comment:
Please remove these links. They will be created automatically on the right side. Provided they are all H2